db/migrate/20230222155842_add_author_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthorToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :author, foreign_key: { to_table: 'users' }
+  end
+end
```

Comment:
- [ ] This table is not required, please 🙏🏽 remove it. We just need user, post, comments, and likes tables.